IARCRP-311: Add tinymce plugins support (#3)
sphericalpm/lektor-tinymce: README.md

    [licence]
    api-key = 'YOUR_API_KEY_HERE'

## Tiny MCE setting

> To past some settings params to the tinymce.init() (plugins for example), just add the following contents to the file `configs/tinymce.ini` in your lektor project folder:

Reviewer: ```suggestion
To pass additional arguments (such as a list of plugins) into [tinymce.init()](https://www.tiny.cloud/docs/configure/integration-and-setup/), just add the following contents to the file `configs/tinymce.ini` in your lektor project folder:
```

    [config]
    settings = 'YOUR_SETTINGS'

Here is the example of the settings string:
    [config]
    settings = "plugins: 'image link lists autoresize table', image_advtab: true,"

> 

Reviewer: ```suggestion

Note that this string is passed directly into tinymce.init(), so it must follow the given format (and include the comma at the end!)
Some plugins, such as image upload, may not be fully integrated with lektor by default.
```
